### DISCUSSION TITLE
AspectRatio adds *:not(style) to child = unwanted Flex?

### DISCUSSION THREAD

**Discussion by lapidus** [chakra-ui/chakra-ui#5344] [Q&A]
### Description

When using AspectRatio, the inner container gets an unwanted Flex declaration 

### Link to Reproduction

https://codesandbox.io/s/recursing-cannon-p82mu?file=/src/index.js

### Steps to reproduce

How can I avoid the "Flex" declaration that gets set on the "SomeContainer"? I would like to stack my items inside the AspectRatio, not flex them.

```
<AspectRatio>
       <Box id="SomeContainer">
          <Box>Row 1</Box>
          <Box>Row 2</Box>
      </Box>
</AspectRatio>
```

![image](https://user-images.githubusercontent.com/66601/148548492-a35524a7-96a8-460a-8840-4a4b7a3c8af3.png)
![image](https://user-images.githubusercontent.com/66601/148548522-d9eac5a7-46cf-432d-910e-a79c66dba442.png)

Thanks!

### Chakra UI Version

1.6.4

### Operating System

- [x] macOS

**Comment by primos63** [comment]
Stacking them is having `flex-direction` set to `column` so simply setting that on `Box` stacks them.
```jsx
        <Box id="SomeContainer" flexDirection='column'>
```

![image](https://user-images.githubusercontent.com/2768849/148581595-562ad20f-d046-4921-b52c-815564f3a545.png)

BTW, this isn't really an issue. Opening a discussion for this would have been better.

**Comment by lapidus** [comment]
Thanks @primos63,

> Opening a discussion for this would have been better.

Any moderator, feel free to convert to a discussion.

My use case is simply to enforce a particular Aspect Ratio for a data visualization component. I was surprised that AspectRatio component applies any kind of styling to contained elements. See this line:
https://github.com/chakra-ui/chakra-ui/blob/main/packages/layout/src/aspect-ratio.tsx#L49

If `flexDirection='column'` is the way to go to reset this, then I'll be happy to apply that.

**Comment by dbousamra** [comment]
Hmm, how do I stop the styles being applied?
